dyninstAPI/src/image.C

```diff
@@ -1613,7 +1612,7 @@ void pdmodule::dumpMangled(std::string &prefix) const
       parse_func * pdf = (parse_func*)*fit;
       if (pdf->pdmod() != this) continue;
 
-      if( ! strncmp( pdf->symTabName().c_str(), prefix.c_str(), strlen( prefix.c_str() ) ) ) {
+      if(pdf->symTabName() != prefix) {
```

Comment:
It seems like this line is comparing to the prefix length instead of full comparison.
Would this be an issue?

Reply:
Derp. You are quite right. I don't know how I didn't see that. The correct substitution would be `if(pdf->symTabName().find(prefix) != 0UL)` which is effective [`std::string::starts_with`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with) from C++20.